CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-project(dnfdragora NONE)
+project(dnfdragora C)
```

Comment:
Well, I changed this to silence the CMake-warning issued by GNUInstallDirs.cmake, assuming every @base buildroot provides gcc.  I can take that commit out, if you prefer.